src/data/contributors/sumitkumar.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Sumit Kumar",
+    "skills": ["Flutter", "Java","Nodejs"],
+    "cause": "Enovations",
+    "image": "Avatar4",
+    "github": "https://github.com/sumitsingh1698",
+    "twitter": "",
```

Comment:
Remove this line, Though I don't know if this could be the point of failure of build process.
If you don't have any twitter handle, then you could just omit writing that line out.